docs/docs-content/integrations/harbor-edge.md

```diff
@@ -209,6 +209,80 @@ docker push 10.10.137.220:30003/spectro-images/alpine:latest
   </TabItem>
   </Tabs>
 
+#### Enable Image Download from Outside of Harbor
+
+When a cluster uses the Harbor Edge-Native Config pack, the cluster assumes all images will be stored in Harbor after
+the initial download. While it's true that all images in the cluster profile for that cluster and managed by Palette
+will be stored in the Harbor registry, images that are not managed by Palette will not exist in the Harbor registry.
+
+This can cause issues if you want to use images that are not managed by Palette in your cluster. Harbor will not store
+those images because they are not part of your cluster profile and were not requested by Palette. However, the Palette
+agent in your Edge host will still try to pull those images from Harbor, resulting in ImagePullBackOff errors.
+
+You can apply the label `stylus.io/imageswap=disable` to a namespace, which instructs the Palette agent to not pull
+images from the Harbor registry. You can do this when you create the namespace, or apply the label to existing
+namespaces. As long as a namespace has the label, the Palette agent will not attempt to pull images in the namespace
+from the Harbor registry.
+
+Use the following steps, which uses Nginx deployment as an example, to label a namespace:
```

Comment:
I'm not sure we need the Nginx example.  I think if we show the user how to add a label to a pack's namespace we can avoid a lot of extra text. yeah they can add the label manually, or they can add it to the cluster profile's pack YAMl where needed. I think showing both patterns, through the CLI with `kubectl label namespace namespace-name stylus.io/imageswap=disable` and the pack YAML config will address the issue. 

https://docs.spectrocloud.com/profiles/profile-customization#pack-labels-and-annotations